:job_id:01000000
:actor_name:XDiTWorker
I0626 11:18:05.583000 2466363 site-packages/torch/distributed/distributed_c10d.py:415] Using backend config: {'cuda': 'nccl'}
[GPU 0] Failed to create pipeline: The provided pretrained_model_name_or_path "/home/shuzuan/prj/ComfyUI-xDit/models/unet/flux/flux1-dev.safetensors" is neither a valid local path nor a valid repo id. Please check the parameter.
Pipeline creation traceback:
Traceback (most recent call last):
  File "/home/shuzuan/prj/ComfyUI-xDit/custom_nodes/comfyui_xdit_multigpu/xdit_runtime/worker.py", line 207, in _create_xfuser_pipeline_if_needed
    self.model_wrapper = xFuserFluxPipeline.from_pretrained(
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/shuzuan/prj/xDiT/xfuser/model_executor/pipelines/pipeline_flux.py", line 64, in from_pretrained
    pipeline = FluxPipeline.from_pretrained(pretrained_model_name_or_path, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/shuzuan/miniconda3/envs/comfyui-xdit/lib/python3.11/site-packages/huggingface_hub/utils/_validators.py", line 114, in _inner_fn
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
  File "/home/shuzuan/miniconda3/envs/comfyui-xdit/lib/python3.11/site-packages/diffusers/pipelines/pipeline_utils.py", line 769, in from_pretrained
    raise ValueError(
ValueError: The provided pretrained_model_name_or_path "/home/shuzuan/prj/ComfyUI-xDit/models/unet/flux/flux1-dev.safetensors" is neither a valid local path nor a valid repo id. Please check the parameter.
⚠️ Pipeline creation failed on GPU 0
